src/Compilers/VisualBasic/Portable/Lowering/ExpressionLambdaRewriter/ExpressionLambdaRewriter.vb

```diff
@@ -735,32 +761,40 @@ lSelect:
-            factoryArgs(1) = _factory.MethodInfo(method, _factory.WellKnownType(WellKnownType.System_Reflection_MethodInfo))
-            Array.Copy(params, 0, factoryArgs, 2, params.Length)
-            Return ConvertRuntimeHelperToExpressionTree("Call", factoryArgs)
+            Return _factory.Convert(
+                _expressionType,
+                    ConvertRuntimeHelperToExpressionTree(
+                        WellKnownMember.System_Linq_Expressions_Expression__Call,
+                        receiver,
+                        _factory.MethodInfo(method, _factory.WellKnownType(WellKnownType.System_Reflection_MethodInfo)),
+                        _factory.Array(_expressionType, params)))
         End Function
 
         ' Emit a Default node for a specific type
         Private Function [Default](type As TypeSymbol) As BoundExpression
-            Return ConvertRuntimeHelperToExpressionTree("Default", _factory.[Typeof](type, _factory.WellKnownType(WellKnownType.System_Type)))
+            Return _factory.Convert(
+                _expressionType,
+                ConvertRuntimeHelperToExpressionTree(WellKnownMember.System_Linq_Expressions_Expression__Default, _factory.[Typeof](type, _factory.WellKnownType(WellKnownType.System_Type))))
         End Function
 
         ' Emit a New node to a specific type with a helper constructor and one argument
         Private Function [New](helper As SpecialMember, argument As BoundExpression) As BoundExpression
-            Return ConvertRuntimeHelperToExpressionTree("New", _factory.ConstructorInfo(helper), argument)
+            Return _factory.Convert(
+                _expressionType,
+                ConvertRuntimeHelperToExpressionTree(WellKnownMember.System_Linq_Expressions_Expression__New_ConstructorInfo_ArrayExpressions,
+                                                     _factory.ConstructorInfo(helper),
+                                                     _factory.Array(_expressionType, argument)))
         End Function
 
         ' Emit a negate node
         Private Function Negate(expr As BoundExpression) As BoundExpression
-            Return ConvertRuntimeHelperToExpressionTree("Negate", expr)
+            Return _factory.Convert(_expressionType, ConvertRuntimeHelperToExpressionTree(WellKnownMember.System_Linq_Expressions_Expression__Negate_Expression, expr))
         End Function
 
         Private Function InitWithParameterlessValueTypeConstructor(type As TypeSymbol) As BoundExpression
             ' The "New" overload without a methodInfo automatically generates the parameterless constructor for us.
             Debug.Assert(type.IsValueType)
-            Return ConvertRuntimeHelperToExpressionTree("New", _factory.[Typeof](type, _factory.WellKnownType(WellKnownType.System_Type)))
+            Return ConvertRuntimeHelperToExpressionTree(WellKnownMember.System_Linq_Expressions_Expression__New_Type, _factory.[Typeof](type, _factory.WellKnownType(WellKnownType.System_Type)))
```

Comment:
It is surprising that we don't need `Convert` here since `InitWithParameterlessValueTypeConstructor` and `[Default]` are used in the same place in the same way.

Reply:
> It is surprising that we don't need `Convert` here since `InitWithParameterlessValueTypeConstructor` and `[Default]` are used in the same place in the same way.

I was adding conversions on as needed basis, when tests were failing. I also wasn't changing all ```ConvertRuntimeHelperToExpressionTree``` calls at the same time, so the conversions were necessary to pass tests at some point. It looks like the need to add conversions for some call sites got eliminated when all call sites were adjusted. I removed those. Thanks.